service/history/task/interface.go

```diff
@@ -129,4 +129,8 @@ const (
 	QueueTypeReplication
 	// QueueTypeCrossCluster is the queue type for cross cluster queue processor
 	QueueTypeCrossCluster
+	// QueueTypeTransfer is the queue type for transfer queue processor
+	QueueTypeTransfer
+	// QueueTypeTimer is the queue type for timer queue processor
+	QueueTypeTimer
```

Comment:
These new enums overlap with existing ones above which is confusing. Do we still need them?

Reply:
Also updated the comments.